Increase dml version (#667)
qbicsoftware/offer-manager-2-portlet: CHANGELOG.rst

**Dependencies**

> * Bump data model lib 2.8.0 -> 2.8.1 and therefore adds 'Flow cell' as unit for service products

Reviewer: ```suggestion
* Bump data-model-lib 2.8.0 -> 2.8.1 which adds 'Flow cell' as unit for service products
```
